lib/account/account-manager.js

```diff
@@ -1,102 +1,171 @@
 "use strict";
+
 /**
  * Manages the business logic of account controls.
  */
 
 /**
- * The object sent in which has information needed to validate the user
- * has properly set up their two factor authentication and has generated
- * a password for use to save.
+ * Contains information needed to validate the user has
+ * properly set up their two factor authentication and
+ * has generated a password.
+ *
+ * @typedef {Object} accountManager~accountinfo
+ * @property {string} regId Unhashed registration id sent to client
+ * @property {string} currentMfa
+ * @property {string} previousMfa
+ * @property {string} password
+ */
+
+/**
+ * Information for a user to generate their password
+ * and validate the account.
  *
- * @typedef {Account~accountinfo}
- * @param {string} accountId this is the unhashed account id sent to client
- * @param {string} currentMfa
- * @param {string} previousMfa
- * @param {string} password
+ * @typedef {Object} accountManager~confirmInfo
+ * @property {string} passwordSalt
+ * @property {string} mfaKey
  */
-class AccountManager {
+
+/**
+ * Contains the registration id needed for a user to
+ * continue on with the sign up process.
+ *
+ * @typeDef {Object} accountManager~completionInfo
+ * @property {string} accountId Unhased account id generated
```

Comment:
"unhased" -> "unhashed" (more than one appearance in these files - search across all files)